### PR TITLE
update the moonbeam development node page

### PR DESCRIPTION
**Pull request by eshaben**
### Description

What started as adding in more content on the `--sealing manual` flag turned into a complete update of the page 😅. Reorganized it in a way that I think makes more sense 

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira

### After Translation Requirements

- [x] No additional PRs are required after the translations are done

### CODE REVIEW

File: builders/get-started/networks/moonbeam-dev.md [hunk 1]
```diff
@@ -10,190 +10,189 @@ description: Follow this tutorial to learn how to set up your first Moonbeam nod
...
+
+If you follow this guide to the end, you will have a Moonbeam development node running in your local environment with 10 [prefunded accounts](#pre-funded-development-accounts) and be able to connect it to the default Polkadot.js GUI.
 
 !!! note
     This tutorial was created using the {{ networks.development.build_tag }} tag of [Moonbase Alpha](https://github.com/PureStake/moonbeam/releases/tag/{{ networks.development.build_tag }}){target=_blank}. The Moonbeam platform and the [Frontier](https://github.com/paritytech/frontier){target=_blank} components it relies on for Substrate-based Ethereum compatibility are still under very active development.
```
Comment (albertov19) [inline builders/get-started/networks/moonbeam-dev.md:18]: Maybe update the Ubuntu version? haha

File: builders/get-started/networks/moonbeam-dev.md [hunk 2]
```diff
@@ -10,190 +10,189 @@ description: Follow this tutorial to learn how to set up your first Moonbeam nod
...
 
-With Polkadot.js Apps connected, you will see the Moonbeam development node waiting for transactions to arrive to begin producing blocks.
+By default, your Moonbeam development node is spun up in instant seal mode, which instantly authors blocks as transactions are received. However, you can specify when blocks should be authored, or sealed, by using the `--sealing` option. 
 
-![Connected to a local node](/images/builders/get-started/networks/moonbeam-dev/moonbeam-dev-7.png)
+The `--sealing` flag accepts any of the following arguments:
 
-## Querying Account State {: #querying-account-state } 
+- `instant` - as we already covered, this is the default option in which blocks are authored as soon as a transaction is received
+- `manual` - allows you to produce blocks manually. If a tranasction is received, a block will not be produced until you manually create one
```
Comment (albertov19) [inline builders/get-started/networks/moonbeam-dev.md:152]:
```suggestion
- `manual` - allows you to produce blocks manually. If a transaction is received, a block will not be produced until you manually create one
```

File: builders/get-started/networks/moonbeam-dev.md [hunk 3]
```diff
@@ -10,190 +10,189 @@ description: Follow this tutorial to learn how to set up your first Moonbeam nod
...
-Click on the top left corner to open the menu to configure the networks, then take the following steps:
+## Configure your Moonbeam Development Node {: #configure-moonbeam-dev-node }
 
-1. Navigate down to open the **Development** sub-menu
-2. Click on **Local Node**, which points Polkadot.js Apps to `ws://127.0.0.1:9944`. If not, you can enter it under **custom endpoint**
-3. Select the **Switch** button, and the site should connect to your Moonbeam development node
+Now that you know how to get a standard Moonbeam development node up and running, you may be wondering how you can configure it. The following sections will cover some common configurations you can use when you spin up your node.
 
-![Select local node](/images/builders/get-started/networks/moonbeam-dev/moonbeam-dev-6.png)
+### Configure Block Production {: #configure-block-production }
```
Comment (albertov19) [inline builders/get-started/networks/moonbeam-dev.md:145]: I do like a section specific to this, but I feel the order is not right.

This is a flag, but then we have a section on `Common Flags to Configure your Node` and then `Common Options to Configure your Node` which also has a section on `--sealing`

File: builders/get-started/networks/moonbeam-dev.md [hunk 4]
```diff
@@ -202,53 +201,117 @@ Flags do not take an argument. To use a flag, add it to the end of a command. Fo
...
 
 ```
 bottom drive obey lake curtain smoke basket hold race lonely fit walk
 ```
 
---8<-- 'code/setting-up-node/dev-accounts.md'
+??? note "Development account addresses and private keys"
+    --8<-- 'code/setting-up-node/dev-accounts.md'
+
+    Also included with the development node is an additional prefunded account used for testing purposes:
```
Comment (albertov19) [inline builders/get-started/networks/moonbeam-dev.md:238]:
```suggestion
Also included with the development node is an additional prefunded account used for testing purposes:
```

Comment (albertov19) [reply builders/get-started/networks/moonbeam-dev.md:238]: If you leave the space this is how it is rendered which is weird:

image

File: builders/get-started/networks/moonbeam-dev.md [hunk 5]
```diff
@@ -202,53 +201,117 @@ Flags do not take an argument. To use a flag, add it to the end of a command. Fo
...
+    ```
+    {{ networks.development.rpc_url }}
+    ```
+
+=== "WSS"
+    ```
+    {{ networks.development.wss_url }}
+    ```
+
+## Connect Polkadot.js Apps to a Moonbeam Development Node {: #connecting-polkadot-js-apps-to-a-local-moonbeam-node }
```
Comment (albertov19) [inline builders/get-started/networks/moonbeam-dev.md:256]: Maybe it is time to take this section out? And just point to the right page?

File: builders/get-started/networks/moonbeam-dev.md (hunk 4 above)
Comment (albertov19) [reply builders/get-started/networks/moonbeam-dev.md:238]: Ahh and please also remove the spaces in the code snippet. So it should be:

```
Also included with the development node is an additional prefunded account used for testing purposes:

--8<-- 'code/setting-up-node/dev-testing-account.md'
```

File: builders/get-started/networks/moonbeam-dev.md (hunk 5 above)
Comment (eshaben) [reply builders/get-started/networks/moonbeam-dev.md:256]: what do you mean the right page?

File: builders/get-started/networks/moonbeam-dev.md (hunk 1 above)
Comment (eshaben) [reply builders/get-started/networks/moonbeam-dev.md:18]: I can do that in a separate PR, as I would have to make sure that all of the pages that use this snippet are all good on Ubuntu 22.04

File: builders/get-started/networks/moonbeam-dev.md (hunk 5 above)
Comment (albertov19) [reply builders/get-started/networks/moonbeam-dev.md:256]: The correct page for polkadot J's apps or maybe we don't show how to connect polkadot J's apps to a network right?

Comment (eshaben) [reply builders/get-started/networks/moonbeam-dev.md:256]: are you referring to a page on our docs site? we have this page but its specific to moonbase alpha: https://docs.moonbeam.network/tokens/connect/polkadotjs/#connecting-to-moonbase-alpha

or are you talking about: https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/democracy?

Comment (albertov19) [reply builders/get-started/networks/moonbeam-dev.md:256]: Maybe this entire section should be removed in favor of pointing people to https://docs.moonbeam.network/tokens/connect/polkadotjs/#connecting-to-moonbase-alpha and revamping that page to add like Moonriver/Moonbeam, wdyt

Comment (eshaben) [reply builders/get-started/networks/moonbeam-dev.md:256]: yeah that sounds good

**Review by albertov19 — APPROVED** [review]
LGTM!! Great!

**Pull request merged**